pkgs/development/libraries/libfive/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, eigen3_3, zlib
+, libpng, boost, qt5, guile
+}:
+
+stdenv.mkDerivation rec {
+  name = "libfive-${version}";
+  version = "755f7fed";
+
+  src = fetchFromGitHub {
+    owner  = "libfive";
+    repo   = "libfive";
+    rev    = "${version}";
+    sha256 = "1cjvmfsjbc3k5zfp22zccmjx0b2rmm2a0vm555f3bqj81yw9v7ps";
+  };  
+  buildInputs = [ cmake pkgconfig eigen3_3 zlib libpng
```

Comment:
`cmake` and `pkgconfig` should be in `nativeBuildInputs` (because they're tools used at build time)

Reply:
Try using ninja with cmake, it often builds faster.

Reply:
How do you mean?

Reply:
Add ninja dependency next to cmake.